### ISSUE TITLE
Cannot download source files from main debian repository: bandwidth exceeded

### ISSUE THREAD

**Issue by ProfessorKaos64** [christopho/solarus#921]
I'm sorry if there was a better way to report this. If you try to download the debs from the debian repository, it says bandwidth exceeded. I'm going to host my own packaging based on the source code. I ran the repo addition script and tried to install the package, but it would not find the package.

At the minimum, can you somehow make available or provide the debian.tar.xz archive so I may see where my build from source is going wrong? I want to provide this engine and some of the games to SteamOS/Jessie users.

Current Debian files: https://github.com/ProfessorKaos64/LibreGeek-Packaging/tree/brewmaster/solarus/debian

**Comment by christopho** [comment]
The Debian repository seems to be down at the moment. @Nate-Devv

**Comment by ProfessorKaos64** [comment]
@christopho thanks for the acknowledgement. The pesky dh_auto_tests are laughing at me. Runs just fine after I hit a shell with pbuilder and run `make -c build/ tests`, but not during the initial run when debhelper is doing things. 

I appreciate the response.

**Comment by Nate-Devv** [comment]
Yeah, the debian repo is having bendwidth issues each month. I can build .debs for solarus 32 and 64 bit fine, but the editor debs won't compile correctly.

**Comment by Nate-Devv** [comment]
I don't think I've ever overriden the dh_auto_build, not sure if that's the problem.

**Comment by ProfessorKaos64** [comment]
I don't want the package installing to /usr/local, that isn't proper. The Arch Linux packages use near the same flags. I'll give it a try anyway soon. I'd love to help host the repository, or Debian packages. I do this for SteamOS / Debian at packages.libregeek.org. If there is a way you can manually / temporarily host debian.tar.xz for the main packages somewhere, that would help me out a lot until bandwidth issues are over with. 

If not, i'll just try to be patient. Thank you for your time @Nate-Devv

**Comment by ProfessorKaos64** [comment]
I see about the GUI, the "gui" folder is cut if you use a release tag. Why is it like this, .gitignore? Why not just implement a default CMAKE flag? If so, it doesn't appear to be documented. I built this without the GUI for now, but I see it is required for the Solarus editor.

**Comment by Nate-Devv** [comment]
I can't get to my other debian.tar.xz files, but here's mine for solarus_editor. I had to change it to a zip.  Keep in mind that I wasn't able to build correctly (editor executable couldn't go to the package.
[solarus-editor_1.4.5-1.debian.zip](https://github.com/christopho/solarus/files/321406/solarus-editor_1.4.5-1.debian.zip)

**Comment by Nate-Devv** [comment]
Also, debian does not install to /usr or /usr/local. dpkg-dbuildpackage chooses it at runtime.  On the packages I've been able to make, the executable has been sent to /usr/games.

**Comment by Nate-Devv** [comment]
That's interesting that the editor requires the gui flag. That could explain why it's not packaging, but I don't think that's documented anywhere. It could be a bug.

**Comment by christopho** [comment]
The -DSOLARUS_GUI is a new flag in 1.5. It is ON by default. The Solarus Launcher GUI needs it (new feature of 1.5) and the editor needs it too.

**Comment by ProfessorKaos64** [comment]
That explains it. I'll just keep to the releases and push out the zsdx game for SteanOS people after testing a little. Thank k you for the response.

**Comment by Nate-Devv** [comment]
Repository bandwith resets on the 13th.

**Comment by Nate-Devv** [comment]
Solarus PPA: https://launchpad.net/~nate-devv/+archive/ubuntu/solarus. Can't get the Editor to compile, so there's no package for it.

**Comment by ProfessorKaos64** [comment]
Thanks for the update nate. I don't mind not having the editor at the moment. I'm not a dev, just handy and learn on the go. I'm not sure I can unravel why the editor doesn't compile, but i'm willing to give it a try. Thanks for updating the PPA (takes care of the bandwidth problem ^_^ )
